.travis.yml

```diff
@@ -1,8 +1,10 @@
 language: node_js
 node_js:
-- 0.10
+  - 0.10
+before_install:
+  - gem install sass
```

Comment:
Can we use `libsass` instead? I *really* don't want to need to have to deal with gems.